project/settings.py

```diff
@@ -90,10 +90,7 @@
 }
 
 # Make this unique, and don't share it with anybody.
-if ENVIRONMENT == 'production':
-    SECRET_KEY = os.environ['DJANGO_SECRET_KEY']
-else:
-    SECRET_KEY = config.get('DJANGO_SECRET_KEY', '')
+SECRET_KEY = config.get('DJANGO_SECRET_KEY', 'gu^&xc)hoibh3x&s+9009jbn4d$!nq0lz+syx-^x8%z24!kfs4')
```

Comment:
Should we continue to use the environment variable here?  It seems like the value is [being set in Puppet](https://github.com/alphagov/govuk-puppet/blob/70b225ec00f96d8c68d787a7a8a3f47e24fa5b07/modules/govuk/manifests/apps/mapit.pp#L81-L83).

Reply:
This caused some confusion at the time, Jenkins was failing due to `django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty`, and after adding the key it resolved the issue.
It also looks like it's [undefined in puppet](https://github.com/alphagov/govuk-puppet/blob/70b225ec00f96d8c68d787a7a8a3f47e24fa5b07/modules/govuk/manifests/apps/mapit.pp#L24) if I'm understanding it correctly

Reply:
>It also looks like it's undefined in puppet if I'm understanding it correctly

There's a value defined in `govuk-secrets` for each environment, e.g. on line 122 of https://github.com/alphagov/govuk-secrets/blob/master/puppet_aws/hieradata/apps/production_credentials.yaml for production.  I'm not really sure what the value is used for, but we probably want to maintain them when upgrading.